src/styles/blocks/catalog.scss

```diff
@@ -1,10 +1,16 @@
+.main {
```

Comment:
It's another block, not `a catalog`. Create a corresponding file.

Reply:
l created a separate file: main-catalog, and in this task it seems to break on smaller sizes, it was not said here that it should look like this on smaller sizes